docs/docs/test-types/executor-gradle.md

```diff
@@ -4,7 +4,13 @@ Testkube allows running Gradle based tasks that could also be tests. For example
 
 * Default command for this executor: `gradle`
 * Default arguments for this executor command: `--no-daemon` `<taskName>` `-p` `<projectDir>`
-(parameters in `<>` are calculated at test execution)
+
+Parameters in `<>` are calculated at test execution:
+
+* `<taskName>` - will be `test` in case of tests of type `gradle/test`, `integrationTest` `gradle/integrationTest` on `gradle/integrationTest`, and empty for `gradle/project`
```

Comment:
```suggestion
* `<taskName>` - Will be `test` in case of tests of type `gradle/test`, `integrationTest` `gradle/integrationTest` on `gradle/integrationTest`, and empty for `gradle/project`.
```